### DISCUSSION TITLE
profile condition=folder/path name

### DISCUSSION THREAD

**Discussion by geextahslex** [mpv-player/mpv#14676] [Q&A] [answered]
Hi, I want to make a profile that detects a word in the root folder. I try to use this `profile-cond=get('media-title', ''):match('Battlestar')~=nil` but this works with the file name and not root/path name.
How to make this work with a folder like this?
"Battlestar.Galactica.S02E01.Die.verlorene.Flotte"

Thank you :)

**Comment by hooke007** [comment]
https://mpv.io/manual/master/#property-list

**Reply by geextahslex** [reply]
Thank you `profile-cond=get('path', ''):match('Battlestar')~=nil` this is working. Is there a way to ignore uppercase/lowercase? because now this is affecting if a word is detected

**Reply by hooke007** [reply]
If my memory serves right, it's lua's limitation.

**Reply by llyyr** [reply]
make the path string all lower or upper case then match with that

**Reply by geextahslex** [reply]
@llyyr sure you can do that, but this is a workaround and you have to do that everytime manually, so not ideal

**Reply by zhongfly** [reply, marked as answer]
use `lower()` or `upper()` convert string to all lower or upper case,then match
example:
`get('path', ''):lower():match('lower')~=nil`
`get('path', ''):upper():match('UPPER')~=nil`

**Reply by geextahslex** [reply]
Nice, that works well `profile-cond=get('path', ''):lower():match('battlestar')~=nil`
Thank you :)
